welcome/index.adoc

```diff
@@ -29,7 +29,7 @@ ifndef::openshift-rosa,openshift-dedicated,openshift-dpu,openshift-telco[]
 To navigate the {product-title} {product-version} documentation, you can use one of the following methods:
 
 * Use the left navigation bar to browse the documentation.
```

Comment:
```suggestion
* Use the navigation bar to browse the documentation.
```

Highly unlikely but just in case the bar is moved to the right or is customizable on docs.redhat.com.

Reply:
Thanks for pointing this out. I had adjusted it, but then lost all my work and had to start over. Style guide states to avoid directionals also.